### PR TITLE
Governance Power BI Report

### PR DESCRIPTION
**Pull request by nteyan**
## 🛠️ Description

The Cloud policy and governance Power BI report enhances the functionality of the Azure Governance workbook. Leveraging the Power BI Resource Graph connector, this report is user-friendly, easy to configure, and adaptable for various purposes.
This dashboard includes : 

- overall resource information,
- virtual machines details,
- managed disks details,
- database information,
- network security group visibility and guidance,
- cost advisor information, and
- policy compliance.

## 📷 Screenshots

image

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [X] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [x] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [ ] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

File: src/power-bi/Governance.Report/definition.pbir
Comment (flanakin) [inline src/power-bi/Governance.Report/definition.pbir:1]: It looks like you enabled the PBIR preview. We don't use enhanced PBIR because it introduces several limitations on end customers, like they can't download or subscribe to reports. As much as I want to use PBIR, I don't think we should use that yet. 

I hate to say this, but I think you'll need to turn the PBIR preview off and then recreate the report. I don't think you can save PBIR into the old format. That said, you should be able to copy/paste the visuals. Sorry about this!

Side note: We can't open this to test it when PBIR is enabled since PBIR is all-or-nothing.
image

File: src/power-bi/Governance.Report/definition/pages/7f44c1a0c828ee16ad2a/page.json
Comment (flanakin) [inline src/power-bi/Governance.Report/definition/pages/7f44c1a0c828ee16ad2a/page.json:1]: This page doesn't belong in a governance report. Can you move this to the Rate optimization report? That can be a separate PR.

File: src/power-bi/Governance.Report/definition/pages/8576f8add1e7014b9e8d/page.json
Comment (flanakin) [inline src/power-bi/Governance.Report/definition/pages/8576f8add1e7014b9e8d/page.json:1]: Can you move this to a separate WorkloadOptimization.pbip report? This can be a separate PR.

File: src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/1e99ea437e1b8b8cd1a8/visual.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        {
+          "properties": {
+            "paragraphs": [
+              {
+                "textRuns": [
+                  {
+                    "value": "The "
+                  },
+                  {
+                    "value": "Governance report",
```
Comment (flanakin) [inline src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/1e99ea437e1b8b8cd1a8/visual.json:25]: Should we set the display name in the report to "Cloud policy and governance" to align with the Framework? Just a thought. I'd keep the file name simple either way. Maybe that's more confusing and not worth it. Curious what others think.

File: src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/cc3c27468cc5454ed6a9/visual.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "visualType": "textbox",
+    "objects": {
+      "general": [
+        {
+          "properties": {
+            "paragraphs": [
+              {
+                "textRuns": [
+                  {
+                    "value": "Governance report",
```
Comment (flanakin) [inline src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/cc3c27468cc5454ed6a9/visual.json:22]: Same question about "Governance" vs. "Cloud policy and governance"

File: src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/e7f58c75a74b8910a661/visual.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+              "expr": {
+                "Literal": {
+                  "Value": "'WebUrl'"
+                }
+              }
+            },
+            "webUrl": {
+              "expr": {
+                "Literal": {
+                  "Value": "'https://aka.ms/finops/toolkit/costsummary'"
```
Comment (flanakin) [inline src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/e7f58c75a74b8910a661/visual.json:103]: Update this link to https://aka.ms/ftk/pbi/governance (already created)

File: src/power-bi/Governance.Report/definition/pages/f6af5a0c453e0b1bced9/page.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/fabric/item/report/definition/page/1.1.0/schema.json",
+  "name": "f6af5a0c453e0b1bced9",
+  "displayName": "Virtual Machine",
```
Comment (flanakin) [inline src/power-bi/Governance.Report/definition/pages/f6af5a0c453e0b1bced9/page.json:4]:
```suggestion
  "displayName": "Virtual machines",
```

File: src/power-bi/Governance.Report/definition.pbir [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "version": "4.0",
+  "datasetReference": {
+    "byPath": {
+      "path": "../Governance.SemanticModel"
```
Comment (flanakin) [inline src/power-bi/Governance.Report/definition.pbir:5]: Use the shared model folder so we can merge cost with other datasets.
```suggestion
      "path": "../Shared.Dataset"
```

**Review by flanakin — CHANGES_REQUESTED** [review]

File: src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/e7f58c75a74b8910a661/visual.json (hunk 3 above)
Comment (nteyan) [reply src/power-bi/Governance.Report/definition/pages/ReportSection04cb7247170034c13d74/visuals/e7f58c75a74b8910a661/visual.json]: Done

File: src/power-bi/Governance.Report/definition/pages/7f44c1a0c828ee16ad2a/page.json
Comment (nteyan) [reply src/power-bi/Governance.Report/definition/pages/7f44c1a0c828ee16ad2a/page.json:1]: I'll do it in separate PR

File: src/power-bi/Governance.Report/definition/pages/8576f8add1e7014b9e8d/page.json
Comment (nteyan) [reply src/power-bi/Governance.Report/definition/pages/8576f8add1e7014b9e8d/page.json:1]: I'll do it in separate PR

File: src/power-bi/Governance.Report/.platform [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/fabric/gitIntegration/platformProperties/2.0.0/schema.json",
+  "metadata": {
+    "type": "Report",
+    "displayName": "CloudPolicyAndGovernance"
```
Comment (flanakin) [inline src/power-bi/Governance.Report/.platform:5]:
```suggestion
    "displayName": "Governance"
```

File: src/power-bi/Governance.Report/report.json
Comment (flanakin) [inline src/power-bi/Governance.Report/report.json:1]: @nteyan I'm adding a comment here since you wanted to make some updates before the PR gets closed. Feel free to resolve this and merge when you're ready. We can update more in a separate PR.

File: docs/_reporting/power-bi/workload-optimization.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+_The Workload optimization report is new and still being fleshed out. We will continue to expand capabilities in each release in alignment with the [Cost optimization workbook](../../_optimize/optimization-workbook/README.md). To request additional capabilities, please [create a feature request](https://aka.ms/ftk/ideas) in GitHub._
+</blockquote>
+
+<br>
+
+## Get started
+
+The **Get started** page includes a basic introduction to the report with additional links to learn more.
+
+> ![Screenshot of the Get started page](https://github.com/user-attachments/assets/68ded074-aa0f-4785-b08d-b072239db903)
```
Comment (flanakin) [inline docs/_reporting/power-bi/workload-optimization.md:51]:
```suggestion
> ![Screenshot of the Get started page](https://github.com/user-attachments/assets/c467d8e2-dd49-4dcf-b5b6-2643a59d57fd)
```

File: src/power-bi/Governance.Report/report.json
Comment (nteyan) [reply src/power-bi/Governance.Report/report.json:1]: Most of the modification requests have been carried out. I'll continue to refine, update, and optimize for the next release.
